src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -48,5 +48,6 @@
   </Assets>
   <Prerequisites>
     <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.DiaSymReader.Native" Version="[15.0,16.0)" DisplayName="Windows PDB reader/writer" />
```

Comment:
The compiler Willow VSIX package can't contain DSRN anymore since the binaries are now installed by msbuild. The compiler Willow package lists all files it contains explicitly. I have [removed](https://github.com/dotnet/roslyn/pull/18928/files#diff-513c6a5492f77871737ce1259f97aad6L37) DSRN from that list. The compiler nuget package should still include DSRN since presumably it should be possible to install the nuget package and run csc from it. That's "outside of msbuild" deployment.